### PR TITLE
Fixed serious grammatical error

### PR DESCRIPTION
**Pull request by LeeConnelly12**
Sounds better.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ More information how to contribute: [https://nextcloud.com/contribute/](https://
 
 Third-party components are handled as git submodules which have to be initialized first. So aside from the regular git checkout invoking `git submodule update --init` or a similar command is needed, for details see Git documentation.
 
-Several apps by default included in regular releases like [firstrunwizard](https://github.com/nextcloud/firstrunwizard) or [gallery](https://github.com/nextcloud/gallery) are missing in `master` and have to be installed manually as required.
+Several apps included by default are regular releases such as [firstrunwizard](https://github.com/nextcloud/firstrunwizard) or [gallery](https://github.com/nextcloud/gallery) are missing in `master` and have to be installed manually as required.
```
Comment (juliushaertl) [inline README.md]: Shouldn't it be: Several apps are included by default in regular app releases ...

**Review by juliusknorr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LeeConnelly12) [reply README.md]: Yeah I agree, sounds better.

Comment (MorrisJobke) [reply README.md]: Fixed in #7298

**Comment by MorrisJobke** [comment]
Closing - see #7298 for this fix.

**Pull request closed without merging**